### PR TITLE
Disable indexing of variables in elasticsearch mappings

### LINKED ISSUE #2665: Investigate how we can store objects (variables) without indexing in elastic
We would like to use json objects/maps instead of a string as `variables` value, see also #2625. If the value of the variables serialized as an json object, then they will be indexed by elastic.
But we would like to avoid the indexing, we need a solution to do that.

blocks #2628 
blocks #2625

### LINKED ISSUE #2681: Complete elasticsearch exporter in default configuration
The elasticsearch exporter configuration in `zeebe.cfg.toml` is missing record types and their default value.

### LINKED ISSUE #2691: Elasticsearch mapping for variable document is invalid
When exporting a variable document elastic rejects the index request, as the mapping is invalid. Currently the test suite is not failing as variable documents are not exported.

```
{"error":{"root_cause":[{"type":"mapper_parsing_exception","reason":"Mapping definition for [document] has unsupported parameters:  [index : true] [store : true]"}],"type":"mapper_parsing_exception","reason":"Failed to parse mapping [_doc]: Mapping definition for [document] has unsupported parameters:  [index : true] [store : true]","caused_by":{"type":"mapper_parsing_exception","reason":"Mapping definition for [document] has unsupported parameters:  [index : true] [store : true]"}},"status":400}
                at org.elasticsearch.client.RestClient$1.completed(RestClient.java:552) ~[elasticsearch-rest-client-6.8.0.jar:6.8.0]
                at org.elasticsearch.client.RestClient$1.completed(RestClient.java:537) ~[elasticsearch-rest-client-6.8.0.jar:6.8.0]
                at org.apache.http.concurrent.BasicFuture.completed(BasicFuture.java:122) ~[httpcore-4.4.11.jar:4.4.11]
                at org.apache.http.impl.nio.client.DefaultClientExchangeHandlerImpl.responseCompleted(DefaultClientExchangeHandlerImpl.java:181) ~[httpasyncclient-4.1.4.jar:4.1.4]
                at org.apache.http.nio.protocol.HttpAsyncRequestExecutor.processResponse(HttpAsyncRequestExecutor.java:448) ~[httpcore-nio-4.4.11.jar:4.4.11]
                at org.apache.http.nio.protocol.HttpAsyncRequestExecutor.inputReady(HttpAsyncRequestExecutor.java:338) ~[httpcore-nio-4.4.11.jar:4.4.11]
                at org.apache.http.impl.nio.DefaultNHttpClientConnection.consumeInput(DefaultNHttpClientConnection.java:265) ~[httpcore-nio-4.4.11.jar:4.4.11]
                at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:81) ~[httpasyncclient-4.1.4.jar:4.1.4]
                at org.apache.http.impl.nio.client.InternalIODispatch.onInputReady(InternalIODispatch.java:39) ~[httpasyncclient-4.1.4.jar:4.1.4]
                at org.apache.http.impl.nio.reactor.AbstractIODispatch.inputReady(AbstractIODispatch.java:114) ~[httpcore-nio-4.4.11.jar:4.4.11]
                at org.apache.http.impl.nio.reactor.BaseIOReactor.readable(BaseIOReactor.java:162) ~[httpcore-nio-4.4.11.jar:4.4.11]
                at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvent(AbstractIOReactor.java:337) ~[httpcore-nio-4.4.11.jar:4.4.11]
                at org.apache.http.impl.nio.reactor.AbstractIOReactor.processEvents(AbstractIOReactor.java:315) ~[httpcore-nio-4.4.11.jar:4.4.11]
                at org.apache.http.impl.nio.reactor.AbstractIOReactor.execute(AbstractIOReactor.java:276) ~[httpcore-nio-4.4.11.jar:4.4.11]
                at org.apache.http.impl.nio.reactor.BaseIOReactor.execute(BaseIOReactor.java:104) ~[httpcore-nio-4.4.11.jar:4.4.11]
                at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor$Worker.run(AbstractMultiworkerIOReactor.java:591) ~[httpcore-nio-4.4.11.jar:4.4.11]
                at java.lang.Thread.run(Thread.java:748) ~[?:1.8.0_222]
Caused by: org.elasticsearch.ElasticsearchException: Elasticsearch exception [type=mapper_parsing_exception, reason=Mapping definition for [document] has unsupported parameters:  [index : true] [store : true]]
        at org.elasticsearch.ElasticsearchException.innerFromXContent(ElasticsearchException.java:509) ~[elasticsearch-6.8.0.jar:6.8.0]
        at org.elasticsearch.ElasticsearchException.fromXContent(ElasticsearchException.java:420) ~[elasticsearch-6.8.0.jar:6.8.0]
        at org.elasticsearch.ElasticsearchException.innerFromXContent(ElasticsearchException.java:450) ~[elasticsearch-6.8.0.jar:6.8.0]
        at org.elasticsearch.ElasticsearchException.failureFromXContent(ElasticsearchException.java:616) ~[elasticsearch-6.8.0.jar:6.8.0]
        at org.elasticsearch.rest.BytesRestResponse.errorFromXContent(BytesRestResponse.java:169) ~[elasticsearch-6.8.0.jar:6.8.0]
        ... 21 more
```

### LINKED ISSUE #2698: Elasticsearch workflow instance creation mapping has outdated property names

### PR DESCRIPTION
**Pull request by menski**
closes #2665 
closes #2691 
closes #2698
closes #2681

### CODE REVIEW

**Review by npepinpe — APPROVED** [review]
👍 

Tested it locally with Kibana looks like it works as expected, i.e. it's not query-able or search-able but you can still view the variables in the source document.

File: exporters/elasticsearch-exporter/src/main/resources/zeebe-record-job-batch-template.json [hunk 1]
```diff
@@ -70,7 +70,7 @@
                   "type": "date"
                 },
                 "variables": {
-                  "type": "text"
```
[the remark this answers is not in the document]
Comment (menski) [reply exporters/elasticsearch-exporter/src/main/resources/zeebe-record-job-batch-template.json]: As far as I know it doesn't matter cause you are not indexing it at all, so they don't care about the type

File: exporters/elasticsearch-exporter/src/main/resources/zeebe-record-incident-template.json [hunk 2]
```diff
@@ -40,7 +40,7 @@
               "type": "long"
             },
             "variables": {
```
[the remark this answers is not in the document]
Comment (menski) [reply exporters/elasticsearch-exporter/src/main/resources/zeebe-record-incident-template.json]: Good point :shrug:

[the remark this answers is not in the document]
Comment (menski) [reply exporters/elasticsearch-exporter/src/main/resources/zeebe-record-incident-template.json]: Actually not, I will remove it from the mapping

**Pull request merged**